common/components/deploy_model/spec.yaml

```diff
@@ -87,6 +87,7 @@ inputs:
       - Standard_NC16as_T4_v3
       - Standard_NC24s_v2
       - Standard_NC24s_v3
+      - Standard_NC24rs_v3
```

Comment:
@novaturient95 - is there a reason why you did not add this your change.

Reply:
It would have been missed. Thanks for adding @esrao